modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -3835,10 +3835,18 @@ namespace fisheye
     @param distorted Output array of image points, 1xN/Nx1 2-channel, or vector\<Point2f\> .
 
     Note that the function assumes the camera intrinsic matrix of the undistorted points to be identity.
-    This means if you want to distort image points you have to multiply them with \f$K^{-1}\f$.
+    This means if you want to distort image points you have to multiply them with \f$K^{-1}\f$ or
+    use another function overload.
      */
     CV_EXPORTS_W void distortPoints(InputArray undistorted, OutputArray distorted, InputArray K, InputArray D, double alpha = 0);
```

Comment:
here K is Knew, but it's called just "K". In the overloaded function below "K" is old K, and there is separate "Knew". I think, it's a bit confusing.